Scripts/Python/xMultiDialogToggle.py

```diff
@@ -0,0 +1,187 @@
+#    Offline KI                                                                #
+#                                                                              #
+#    Copyright (C) 2004-2011  The Offline KI contributors                      #
+#    See the file AUTHORS for more info about the contributors (including      #
+#    contact information)                                                      #
+#                                                                              #
+#    This program is free software: you can redistribute it and/or modify      #
+#    it under the terms of the GNU General Public License as published by      #
+#    the Free Software Foundation, either version 3 of the License, or         #
+#    (at your option) any later version, with or (at your option) without      #
+#    the Uru exception (see below).                                            #
+#                                                                              #
+#    This program is distributed in the hope that it will be useful,           #
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of            #
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the             #
+#    GNU General Public License for more details.                              #
+#                                                                              #
+#    Please see the file COPYING for the full GPLv3 license, or see            #
+#    <http://www.gnu.org/licenses/>                                            #
+#                                                                              #
+#    Uru exception: In addition, this file may be used in combination with     #
+#    (non-GPL) code within the context of Uru.                                 #
+#                                                                              #
+#==============================================================================#
+
+from Plasma import *
+from PlasmaTypes import *
+import PlasmaControlKeys
+import string
+Activate = ptAttribActivator(1, ' clickable ', netForce=1)
+Vignettes = ptAttribString(4, 'Toggle multiple dialogs - by Name')
+SingleUser = ptAttribBoolean(5, 'One user only?', 0)
+KeyMap = {}
+KeyMap[PlasmaControlKeys.kKeyMoveForward] = PlasmaControlKeys.kKeyCamPanUp
+KeyMap[PlasmaControlKeys.kKeyMoveBackward] = PlasmaControlKeys.kKeyCamPanDown
+KeyMap[PlasmaControlKeys.kKeyRotateLeft] = PlasmaControlKeys.kKeyCamPanLeft
+KeyMap[PlasmaControlKeys.kKeyRotateRight] = PlasmaControlKeys.kKeyCamPanRight
+kExit = 99
+Vignette = None
+class xMultiDialogToggle(ptModifier,):
+
+    def __init__(self):
+        ptModifier.__init__(self)
+        self.id = 5104999
+        self.version = 1
+        minor = 0
+        self.me = self.__class__.__name__
+        self.VignetteList = []
+        print ('__init__%s v. %d.%d' % (self.me, self.version, minor))
+
+
+
+    def IGetAgeFilename(self):
```

Comment:
```suggestion
        print ('__init__%s v. %d.%d' % (self.me, self.version, minor))

    def IGetAgeFilename(self):
```
Having so many extra lines of whitespace makes me and my not-huge monitor sad, haha. Please only one line in between blocks.